tests/unit/environments/test_environments_builders.py

```diff
@@ -83,6 +83,38 @@ def test_build_environment_model_with_name():
     assert environment_model.name == environment_name
 
 
+def test_build_environment_model__project_has_server_key_only_feature_ids__return_expected() -> (
+    None
+):
+    # Given
+    environment_name = "some_environment"
+    environment_dict = {
+        "id": 1,
+        "api_key": "api-key",
+        "name": environment_name,
+        "project": {
+            "id": 1,
+            "name": "test project",
+            "organisation": {
+                "id": 1,
+                "name": "Test org",
+                "stop_serving_flags": False,
+                "persist_trait_data": True,
+                "feature_analytics": True,
+            },
+            "hide_disabled_flags": False,
+            "server_key_only_feature_ids": [1],
+        },
+        "feature_states": [],
+    }
+
+    # When
+    environment_model = build_environment_model(environment_dict)
+
+    # Then
+    assert environment_model.project.server_key_only_feature_ids == [1]
```

Comment:
I get that it's quite simple here, but I'd rather than we pulled this out as a variable rather than defining the list `[1]` twice.

Reply:
Fair point, done.